ECOSYSTEM.md

```diff
@@ -4,8 +4,6 @@ This is a list of axios related libraries and resources. If you have a suggestio
 
 ## Libraries
 
-### General
-
```

Comment:
@simoneb Damn it! I did something wrong when fixing conflicts. Would you mind help recover this remove?

Reply:
done